src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -535,6 +536,7 @@ public override void _Ready()
         atpConsumptionBar.SelectedType = SegmentedBar.Type.ATP;
 
         RegisterTooltips();
+        UpdateMicrobePartSelections();
```

Comment:
Like here, it's already almost done the split with the init and update for the part selection buttons.